src/main/java/com/github/dockerjava/core/command/CreateContainerCmdImpl.java

```diff
@@ -473,6 +473,12 @@ public String getPidMode() {
     }
 
     @Override
+    @JsonIgnore
+    public Long getShmSize() {
+        return hostConfig.getShmSize();
```

Comment:
Why are you adding it to create if it was already added to hostconfig? Fill HostConfig and attach to CreateContainer.